recipe/patches/0012-avoid-deprecated-find_package-CUDA-in-caffe2-CMake-m.patch

```diff
@@ -262,7 +262,7 @@ index 781a4e6819f..0c151f51c6c 100644
 +      set(add_ptx TRUE)
 +      set(arch_name ${CMAKE_MATCH_1})
 +    endif()
-+    if(arch_name MATCHES "^([0-9]\\.[0-9](\\([0-9]\\.[0-9]\\))?)$")
++    if(arch_name MATCHES "^(1?[0-9]\\.[0-9](\\([0-9]\\.[0-9]\\))?)$")
```

Comment:
This is the only actually relevant change in 0d06933c6407b4cccef78c092044c1dbb136e0a0